kubesae/providers/gcp.py

```diff
@@ -0,0 +1,52 @@
+"""GCP provider module.
+
+Provides helpful utilities for working with kubernetes and the Google Container Registry.
+"""
+
+import invoke
+from colorama import Style
+
+
+@invoke.task()
+def gcp_docker_login(c):
+    """
+    Authenticate into GCP, and configure Docker.
+
+    Usage: inv gcp.docker-login
+
+    Config:
+
+        app: Name of the project in GCP
+        repository: Name of docker repository, e.g. us.gcr.io/myproject/myproject
+    """
+    registry = c.config.repository.split("/")[0]
+    c.run("gcloud auth login")
+    c.run(f"gcloud config set project {c.config.app}")
+    c.run(f"gcloud auth configure-docker {registry}")
+
+
+@invoke.task()
+def configure_kubeconfig(c, cluster=None, region=None):
```

Comment:
Can this be named ``configure-gcp-kubeconfig`` for consistency with ``configure-eks-kubeconfig``?

Reply:
Sure, that can be done